tests/acceptance/features/apiSharingNg1/enableDisablePermissionsRole.feature

```diff
@@ -0,0 +1,401 @@
+@env-config
+Feature: enable disable permissions role
+  As a user
+  I want to enable/disable permissions role on shared resources
+  So that I can control the accessibility of shared resources to sharee
+
+  Background:
+    Given these users have been created with default attributes:
+      | username |
+      | Alice    |
+      | Brian    |
+
+
+  Scenario Outline: users list the shares shared with Secure Viewer after the role is disabled (Personal Space)
+    Given the administrator has enabled the permissions role "Secure Viewer"
+    And user "Alice" has uploaded file with content "some content" to "textfile.txt"
```

Comment:
maybe we should assert these in a single scenario for these kinds of tests (edge-cases):
- sharedByMe (sharer)
- sharedWithMe (sharee)
- list me/drives (sharee)
- PROPFIND Shares space (sharee)
- some action by sharee (e.g.: download file)

Reply:
I have updated the changes in the follow up PR https://github.com/owncloud/ocis/pull/10756. We can merge this PR and add these changes in other PR.